manifests/profiles/PROFILES.md

```diff
@@ -10,7 +10,7 @@ will provide further instructions. Additionally, please read the RFC requirement
 
 ## Vendor optimized profiles
 
-| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
-| -------------------- | ------------- | --------------- | ---------------- |
-| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
-| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
+|Profile Name | K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
+|-------------| -------------------- | ------------- | --------------- | ---------------- |
+|openshift| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
+|| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
```

Comment:
I don't see a profile for Kubernetes. What should this be called? Also, is discuss.istio.io the actual maintainer here?